### DISCUSSION TITLE
New discussion link leads to 404 if user is logged out

### DISCUSSION THREAD

**Discussion by ondrejmirtes** [community/community#2890] [Discussions] [answered]
I've wanted to add a Support link on my website that would lead directly to `https://github.com/phpstan/phpstan/discussions/new?category=support`. But it shows 404 to logged-out GitHub users. I think it should be a nice experience, prompting users to log in, because this just looks like I made a mistake and linked to a wrong page.

Thank you.

**Comment by becca** [comment, marked as answer]
Duplicate of https://github.com/github/feedback/discussions/29. We are tracking this work in our backlog and will let y'all know when we 🚢  a fix/update for this.
